### ISSUE TITLE
Test on kinetics 400

### ISSUE THREAD

**Issue by manhcntt21** [Sense-X/UniFormer#86]
Hi, thank for your implementation.

I have read your paper, and now just want to run test, I followed the steps here

![image](https://user-images.githubusercontent.com/27800932/197446741-e3bf0002-023c-42a7-a7c0-fa410ad399c2.png)

I downloaded pre-trained on [link](https://drive.google.com/file/d/1-kitVTkWErHXI_x-sLItvK9-sOmb53j6/view)

According to my limited knowledge, run pre-trained will not need dataset, is that true?

Some changes I made:

1.  I created path_to_models folder inside video_classification folder
2.  I changed _C.DIST_BACKEND = "gloo", because I'm windows user (nccl -> gloo)
3.  I changed NUM_GPS 1 in test.sh and test.yaml
4.  PRETRAIN_NAME: 'uniformer_small_k400_8x8'  on test.yaml

I changed some config and run 'bash ./exp/uniformer_s8x8_k400/test.sh, error in your_path_to_data

![image](https://user-images.githubusercontent.com/27800932/197448103-e8be0cb4-784b-49e3-a3f5-a77ca4d4ae05.png)

So, my question is there a way to run test without downloading the data kinetics 400?, because my computer does not enough memory.

I would appreciate it if you have detailed instruction. 

Thank you so much

**Comment by Andy1621** [comment]
Thanks for your question! 
What do you want to do for running the test? 
- If you want to check the model, I suggest you use the model code in [hugging_face](https://huggingface.co/spaces/Sense-X/uniformer_video_demo/blob/main/uniformer.py)
- If you want to check the codebase, you can create a tiny dataset with a few videos. You only need to prepare some CSV files as [here](https://github.com/Sense-X/UniFormer/tree/main/video_classification/data_list/k400).

**Comment by manhcntt21** [comment]
Thanks for your reply, I'm a graduate student, so I want to check the codebase and then conduct my own experiments. Thanks for your advice, I will try and provide my results for you.

**Comment by manhcntt21** [comment]
To continue the learning process, I download a video on kinetics-400 test in  **https://s3.amazonaws.com/kinetics/400/test/part_0.tar.gz**

And, then I insert its name into the file test.csv as below, number 1 is random number I added, because I don't know label what its label is

![image](https://user-images.githubusercontent.com/27800932/197832281-2a78cb74-168a-4ad8-b3cb-b71d756ba424.png)

**Comment by Andy1621** [comment]
You have correctly run the codebase!
Since your want to conduct your experiments, the first step is to communicate with your advisor about a research topic.

If you want to apply UniFormer for a specific domain, you may need to prepare your own dataset and fine-tune the pretrained models we provided!

**Comment by manhcntt21** [comment]
Thank you very much!
